source/crud/transaction.txt

```diff
@@ -163,6 +163,8 @@ completes the following actions:
 
 .. sharedinclude:: dbx/transactions-parallelism.rst
 
+   .. replacement:: driver-specific-content
```

Comment:
S: this is getting fixed in my bulkwrite pr #250 but good to remove the error anyways